### PR TITLE
gnucash: Simplify

### PR DESCRIPTION
**Pull request by okurz**

### CODE REVIEW

**Comment by asmorodskyi** [comment]
Proof run ?

File: tests/x11/gnucash.pm [hunk 1]
```diff
@@ -22,19 +22,18 @@ sub run() {
...
-                          # Leave tips windows for GNOME case
-    if (check_var("DESKTOP", "gnome") || get_var("DESKTOP") eq "xfce") { sleep 3; send_key "alt-c"; }
-    wait_idle;
-    send_key "ctrl-q";    # Exit
-
-    if (check_screen("gnucash-save-changes", 10)) {
-        send_key "alt-w";    # Close _Without Saving
+    assert_screen 'test-gnucash-2';
+    # Leave tutorial window
+    wait_screen_change { send_key 'alt-f4' };
```
Comment (asmorodskyi) [inline tests/x11/gnucash.pm]: you suppose to cause this screen change after previous assert ?

File: tests/x11/gnucash.pm [hunk 2]
```diff
@@ -22,19 +22,18 @@ sub run() {
...
+    # Leave tips windows for GNOME case
+    if (check_var('DESKTOP', 'gnome') || check_var('DESKTOP', 'xfce')) {
+        wait_screen_change { send_key 'alt-c' };
     }
+    send_key 'ctrl-q';    # Exit
+    assert_screen [qw(gnucash-save-changes generic-desktop)];
+    wait_screen_change { send_key 'alt-w' } if match_has_tag 'gnucash-save-changes';
```
Comment (asmorodskyi) [inline tests/x11/gnucash.pm]: I'm lost we already exit application , what suppose to catch this keys ?

File: tests/x11/gnucash.pm (hunk 1 above)
Comment (okurz) [inline tests/x11/gnucash.pm]: it's the other way around: 'alt-f4' is sent, then os-autoinst should wait for screen change

File: tests/x11/gnucash.pm (hunk 2 above)
Comment (okurz) [inline tests/x11/gnucash.pm]: we did not exit yet if it shows "gnucash-save-changes" instead

**Comment by okurz** [comment]
proof run: http://lord.arch/tests/6111

**Pull request merged**